### PR TITLE
Safe 4 updates

### PR DESCRIPTION
**Pull request by teknikal-wizard**

### CODE REVIEW

File: docs/component-fable.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ It also provides *rich* integration with the JS ecosystem which means that you c
 
 ## How does Fable integrate with SAFE?
 
-Fable is a .NET tool that generates JavaScript files directly this allows us to write full front end applications using F#. Being able to write both the Server and Client in the same language offers huge benefits especially when you can share code between the two, without the need for duplication. More information on code sharing can be found [here](../feature-clientserver).
+Fable is a tool that generates JavaScript files from .NET code. This allows us to write full front end applications using F#. Being able to write both the Server and Client in the same language offers huge benefits especially when you can share code between the two, without the need for duplication. More information on code sharing can be found [here](../feature-clientserver).
```
Comment (mattgallagher92) [inline docs/component-fable.md:12]:
```suggestion
Fable is a tool that generates JavaScript files from F# code. This allows us to write full front end applications using F#. Being able to write both the Server and Client in the same language offers huge benefits especially when you can share code between the two, without the need for duplication. More information on code sharing can be found [here](../feature-clientserver).
```
I'm pretty sure it's only from F# code - Fable is enabled by F# Compiler services, so is F#-specific and .NET is not really involved at all.

File: docs/recipes/upgrading/v3-to-v4.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+**Important** If you have made any modifications to the webpack file, you will need to apply them back in manually.
+
+#### 11. Update TargetFramework in all projects
+* **Overwite** the `Client.Tests.fsproj` file
+* **Overwite** the `Server.Tests.fsproj` file
+* **Overwite** the `Shared.Tests.fsproj` file
+* **Overwite** the `Client.fsproj` file
+* **Overwite** the `Server.fsproj` file
+* **Overwite** the `Shared.fsproj` file
```
Comment (mattgallagher92) [inline docs/recipes/upgrading/v3-to-v4.md:85]: Do we need to say about copying across any modifications to the file? Would it be better to get them just to change the TargetFramework property?

**Review by mattgallagher92 — CHANGES_REQUESTED** [review]
Looks good, I've suggested a minor tweak or two.

File: docs/recipes/upgrading/v3-to-v4.md (hunk 2 above)
Comment (teknikal-wizard) [reply docs/recipes/upgrading/v3-to-v4.md:85]: Yeah perhaps - I was just really following the precedent from the 2->3 migration of either remove or replace file instructions, with caveats that if you have edited the originals, you need to take that into consideration.

I'd need to check that doing that is all that is necessary but sounds feasible

Comment (teknikal-wizard) [reply docs/recipes/upgrading/v3-to-v4.md:85]: Just checked the diff and it is the only change in the files so yeah, I'll do that

File: docs/recipes/upgrading/v3-to-v4.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# How do I upgrade from SAFE v2 to v3?
```
Comment (isaacabraham) [inline docs/recipes/upgrading/v3-to-v4.md:1]: Title needs updating.

**Review by isaacabraham — APPROVED** [review]
LGTM with just a tiny change needed.

**Pull request merged**